src/models/User.js

```diff
@@ -201,6 +201,13 @@ export default (sequelize, DataTypes) => {
     user.password = await CryptData.encryptData(user.password);
   });
 
+  User.associate = models => {
+    User.hasMany(models.Request, {
+      foreignKey: 'userId',
+      onDelete: 'CASCADE'
+    });
+  };
+
```

Comment:
We dont need this association definition since the association between users n request table has been defined already.